src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesTestUtil.java

```diff
@@ -72,8 +73,12 @@ public static KubernetesCloud setupCloud() throws UnrecoverableKeyException, Cer
         if (StringUtils.isNotBlank(branch)) {
             String namespaceWithBranch = String.format("%s-%s", DEFAULT_TESTING_NAMESPACE, branch);
             LOGGER.log(FINE, "Trying to use namespace: {0}", testingNamespace);
-            if (client.namespaces().withName(namespaceWithBranch).get() != null) {
-                testingNamespace = namespaceWithBranch;
+            try {
+                if (client.namespaces().withName(namespaceWithBranch).get() != null) {
+                    testingNamespace = namespaceWithBranch;
+                }
+            } catch (KubernetesClientException e) {
+                // nothing to do
```

Comment:
@alorlea Hi. Just curious but shouldn't you just fail-fast in this case, rather than suppressing a KubernetseClientException like this?